enterprise/models.py

```diff
@@ -473,6 +478,17 @@ def unlink_user(self, enterprise_customer, user_email):
             # not capturing DoesNotExist intentionally to signal to view that link does not exist
             link_record = self.get(enterprise_customer=enterprise_customer, user_id=existing_user.id)
             link_record.delete()
+
+            # Remove the SailThru flags for enterprise learner.
+            if update_user:
```

Comment:
I think it would be better to add a check for `update_user` and raise `NotConnectedToOpenEdX` instead of this condition.

```
if update_user is None:
      raise NotConnectedToOpenEdX("This package must be installed in an OpenEdX environment.")
```